src/SourceBrowser/src/SourceIndexServer/Helpers.cs

```diff
@@ -3,93 +3,27 @@
 using System.Net.Http;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.SourceBrowser.SourceIndexServer.Models;
 
 namespace Microsoft.SourceBrowser.SourceIndexServer
 {
     public static class Helpers
     {
-        public static async Task ProxyRequestAsync(this HttpContext context, HttpClient client, string targetUrl, Action<HttpRequestMessage> configureRequest = null)
+        public static async Task ProxyRequestAsync(this HttpContext context, string targetPath, Action<HttpRequestMessage> configureRequest = null)
         {
-            using (var req = new HttpRequestMessage(HttpMethod.Get, targetUrl))
+            var fs = new AzureBlobFileSystem(IndexProxyUrl);
+            context.Response.Headers.Append("Content-Type", "text/html");

```

Comment:
Is this the right content type for the `.txt` extension as well? https://github.com/dotnet/source-indexer/pull/154/files#diff-afc620faafd6240892798b2e38ce89db157cc19ea9ef02bd9ebd005edce99b44R33  Could be, just checking.  I'm not sure what the index serves up that's actually a txt, honestly.

I think we could see what the existing site does and examine the headers in the browser F12 tools.

Reply:
Using text/html for .txt doesn't break anything, though yes it's incorrect. Mostly the .txt stuff is debug stuff e.g. Errors.txt shows a log of errors from index generation.